GeorgianMemenator/Network/Mappers/DogBreedsMapper.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  DogBreedsMapper.swift
+//  GeorgianMemenator
+//
+//  Created by Qeti Avanashvili on 25.02.23.
+//
+
+import Foundation
+
+enum DogBreedsMapper {
+    static func toEntity(dto: DogBreedDTO) -> DogBreed {
+        .init(
+            name: dto.attributes.name,
+            description: dto.attributes.description,
+            minLife: Int(dto.attributes.life.min),
```

Comment:
There is no need to cast integers to Int, so you can safely remove `Int(dto.att...)`, just pass the `dto.attributes.life.min`. Its for all properties where you are casting them to Int.